.github/workflows/nightly-ci.yml

```diff
@@ -0,0 +1,126 @@
+name: test
```

Comment:
What do you think about naming this file something like `nightly-tfe-ci.yml` so then it's clearer in the GH workflows UI what the workflow is for?